docs/docs/0.7.2/documentation/security.md

```diff
@@ -75,6 +110,17 @@ If you turn off authentication, any dataplane can request certificate for any se
 :::
 ::::
 
+#### Multizone
+
+When running in multizone mode you can generate tokens both on Global and Remote Control Plane
+If you configure your deployment pipeline to generate Dataplane Token before running a dataplane you can use Remote CP. This way Global CP is not a single point of failure.
```

Comment:
```suggestion
If you configure your deployment pipeline to generate data plane tokens before running a data plane, you can use Remote CP. This way Global CP is not a single point of failure.
```
Would like this to be slightly expanded on since I am slightly confused. May cause confusion for external folks too.

Reply:
To be honest I'm not sure how to extend this section 🤔 
If you have a setup that you generate dp token before you run the kuma-dp then if you do this on global and global is down, you cannot spawn a new DP. If you do this on remote and global is down you are still fine with spawning new dps